src/main/java/mods/eln/transparentnode/transformer/TransformerDescriptor.java

```diff
@@ -83,22 +83,22 @@ void draw(Obj3DPart fero, int priCableNbr, int secCableNbr, boolean hasCasing, f
             fero.draw();
             if (priCableNbr != 0) {
                 GL11.glPushMatrix();
-                float y = (priCableNbr - 1) * 1f / 16f;
-                GL11.glTranslatef(0f, -y, 0f);
+                float separation = 0.5f / (priCableNbr + 1);
+                GL11.glTranslatef (0f, separation-4/16f, 0f);
                 for (int idx = 0; idx < priCableNbr; idx++) {
                     sbire.draw();
-                    GL11.glTranslatef(0f, 2f / 16f, 0f);
+                    GL11.glTranslatef (0f, separation, 0f);
                 }
                 GL11.glPopMatrix();
             }
             if (secCableNbr != 0) {
                 GL11.glPushMatrix();
                 GL11.glRotatef(180, 0f, 1f, 0f);
-                float y = (secCableNbr - 1) * 1f / 16f;
-                GL11.glTranslatef(0f, -y, 0f);
+                float separation = 0.5f / (secCableNbr + 1);
```

Comment:
A call to glScalef around here might accomplish two things:

- Thin out the models on the Y axis enough that they remain visually distinct at higher numbers;
- Get rid of this slightly complicated math to calculate a separation--the separation can be counted in the same units as the Y-width (height?) of the model itself :)

On that note, I wouldn't mind the stacking limit as being the limit for a transformer, at least for experimentation. To assuage concerns of tier-skipping, I'd rather introduce a config or make them undesirable at high ratios for some reason (e.g., they become lossy, they have high capacitance, etc.).

Reply:
(NB: I don't know where your transform origin is, and that *might* have an undesirable effect of needing to multiply various translate parameters in this method :)

Reply:
Already tried that; the existing wires (called sbire model in the code) are introduced as a regular object from blender, and pre-translated.  We'd have to un-translate first at and that point it's better to scrap it.

Reply:
I'm amiable to the "scrap it" route, it's definitely simple enough to be practical to draw right in the code :)